src/core/Pricing/PricingCards.tsx

```diff
@@ -192,7 +186,7 @@ const PricingCards = ({ data, delimiter }: PricingCardsProps) => {
                         href={cta.url}
                         onClick={cta.onClick}
                         disabled={cta.disabled}
-                        rightIcon="icon-gui-arrow-long-right-outline"
+                        rightIcon="icon-gui-arrow-right-outline"
```

Comment:
This arrow looks different to the design, but I think that's a problem with the design - you've used the appropriate Heroicon so I wouldn't change this